src/glib.mk

```diff
@@ -102,7 +102,7 @@ define $(PKG)_BUILD
     ln -sf '$(PREFIX)/$(BUILD)/bin/glib-compile-resources' '$(PREFIX)/$(TARGET)/bin/'
 
     # cross build with posix threads
-    $(MXE_MESON_WRAPPER) --buildtype=release -Dtests=false -Dthreads=true '$(BUILD_DIR)' '$(SOURCE_DIR)' && \
+    $(MXE_MESON_WRAPPER) --buildtype=release -Dtests=false -Dforce_posix_threads=true '$(BUILD_DIR)' '$(SOURCE_DIR)' && \
```

Comment:
ow apparently my `threads=true` flag was not good enough. I was already wondering if it was posix related.